utf8 error while running tests

from jenkins jobs (not sure if this causes the tests to fail)
```
...
[INFO] Compiling 159 source files to /var/lib/jenkins/workspace/panc/panc/target/test-classes
[ERROR] /var/lib/jenkins/workspace/panc/panc/src/test/java/org/quattor/pan/dml/functions/IpToLongTest.java:[2,36] unmappable character for encoding UTF-8
[ERROR] /var/lib/jenkins/workspace/panc/panc/src/test/java/org/quattor/pan/dml/functions/IpToLongTest.java:[2,43] unmappable character for encoding UTF-8
[INFO] /var/lib/jenkins/workspace/panc/panc/src/test/java/org/quattor/pan/dml/operators/SimpleListVariableTest.java: Some input files use or override a deprecated API.
...
```

--- a/panc/src/test/java/org/quattor/pan/dml/functions/IpToLongTest.java
+++ b/panc/src/test/java/org/quattor/pan/dml/functions/IpToLongTest.java
@@ -1,5 +1,5 @@
 /*
-  Copyright 2013 - Luis Fernando Muñoz Mejías and Universiteit Gent.
+  Copyright 2013 - Luis Fernando MuÃ±oz MejÃ­as and Universiteit Gent.
 
   Licensed under the Apache License, Version 2.0 (the "License"); you
   may not use this file except in compliance with the License.  You

